utils/util.go

```diff
@@ -60,3 +60,30 @@ func ToHealthStatus(s string) types.HealthStatus {
 
 	return types.UNKNOWN_HEALTH_STATUS
 }
+
+func SetStepData(stepData map[interface{}]interface{}, startTime, endTime int64, stepName, status string) {
```

Comment:
Please add some method level comments describing the input/output and the function responsibility. It will be go-doc friendly as well.